### ISSUE TITLE
Can't insert a new entry in my database

### ISSUE THREAD

**Issue by Diamax10** [btk5h/skript-db#8]
Hello,

I have a problem when I want to do an "INSERT INTO" query to my SQL server it does nothing. I've tested with 1.11.2 - 1.12.1 - 1.12.2 Minecraft's version but it does not work :/ I can do a "SELECT" query it works but not an insert ... In the past it worked perfectly (in another server).
No errors in the console. I tested my server's mysql user and root user.

**Informations:**
Skript dev35b (tested with dev34)
Skript code : 
``execute "INSERT INTO `players` (`name`, `uuid`, `rank`, `works`, `friend_invitation`, `register_date`, `last_seen`, `last_ip`) VALUES (%name of player%, %UUID of player%, 'Joueur', '', 'allow', %now%, %now%, %ip of player%)" in {sql}``

Plugins: Skript, skript-db, skript-mirror

Thanks for your help :)

**Comment by btk5h** [comment]
Does `the last database error` contain anything after executing that statement?

**Comment by Diamax10** [comment]
Yes, it says : ``Invalid argument value: java.io.NotSerializableException``

**Comment by Diamax10** [comment]
Hello, do you have anything new about the problem ?

If I execute SQL command in CLI it works perfectly

**Comment by btk5h** [comment]
Skript's date class cannot be stored directly in a database. I'll add a way to convert Skript dates into SQL dates.

**Comment by Diamax10** [comment]
Yeah, it works perfectly when I convert date to text :)

**Comment by Hakim567** [comment]
any updates?
